core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -132,14 +132,14 @@ void addChildContext(BaseEventContext childContext) {
   @Override
   public final void success() {
     if (isResponseDone()) {
-      if (LOGGER.isDebugEnabled()) {
-        LOGGER.debug(this + " empty response was already completed, ignoring.");
+      if (debugLogEnabled) {
+        LOGGER.debug("{} empty response was already completed, ignoring.", this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Invoke method(s) only conditionally. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLjRWWotIXnmsUmgkx&open=AWkLjRWWotIXnmsUmgkx)

[ISSUE_KEY]: AWkLjRWWotIXnmsUmgkx